src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -575,6 +608,16 @@ public String getImageTopLevelType() {
         return imageTopLevelType;
     }
 
+    public String getAvailabilityType(AvailabilityTypeClass availabilityTypeClass) {
+        if (availabilityTypeClass == null) {
```

Comment:
redundant

Reply:
Need this check here, otherwise `availabilityTypeClass.getAvailabilitySet()` will throw nullpointerException.

Reply:
Need this check here, otherwise `availabilityTypeClass.getAvailabilitySet()` will throw nullpointerException.